build-parent/pom.xml

```diff
@@ -77,7 +77,7 @@
 
     <version.flyway>4.2.0</version.flyway>
 
-    <version.gradle>2.4</version.gradle>
+    <version.gradle>3.0</version.gradle>
```

Comment:
The Gradle tooling dependency was already at 3.0. I am ensuring that we make use of the same version across.